README.md

```diff
@@ -0,0 +1,50 @@
+# VCNL4040
+
+Low level driver for the VCNL4040 sensor.
```

Comment:
Nit. And since you are the second one not doing it this way, maybe I should stop and change the "guidelines".

```suggestion
A low level driver for the VCNL4040 sensor.
```